client/src/components/_common/Pill/Pill.module.scss

```diff
@@ -1,11 +1,13 @@
-.pill {
-  padding: 0 1em; /* buffer retained on 2+ digit badge, without increase on 1 digit badge */
-  min-width: 1.3em;  /* ~18px design * 1.2 design-to-app ratio (rounded) */
+/* TODO: Consider sharing code between (History)Badge and Pill */
+.root {
+  font-size: 0.75rem; /* 12px (10px design * 1.2 design-to-app ratio) */
```

Comment:
An ancestor element may have `font-size` set. That likely should not affect pills. And if it should, the author of the ancestor element's CSS can explicitly change the pill, after which we can decide (using the newly provided use case) whether to (a) apply via ancestor CSS, (b) always allow cascading of font-size onto Pill, or (c) integrate cascading as an option for Pill.